.php-cs-fixer.dist.php

```diff
@@ -51,9 +51,6 @@
                 'yield',
             ],
         ],
-        'braces' => [
-            'allow_single_line_anonymous_class_with_empty_body' => true,
-        ],
```

Comment:
See https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/blob/v3.16.0/src/Fixer/Basic/BracesFixer.php#L211-L235 for how the maintainers of `friendsofphp/php-cs-fixer` replaced the configuration of the `braces` fixer by proxying to newly created fixers.